microsoft-edge/developer/index.yml

```diff
@@ -182,7 +182,7 @@ landingContent:
           - text: View formatted JSON responses or files
             url: ../devtools-guide-chromium/json-viewer/json-viewer.md
 
-          - text: Sign up for the Ad Selection API
+          - text: Sign up for the Ad Selection API  # todo: moved to "Experimental web platform features" toc bucket, delete?
```

Comment:
Delete "Sign up for the Ad Selection API" from "Edge-specific web development tips" card?
Article moved to "Experimental web platform features" toc bucket.